templates/bare/test/bare.js

```diff
@@ -0,0 +1,52 @@
+const { APP_IDS } = require('@aragon/templates-shared/helpers/apps')
+const { getEventArgument } = require('@aragon/test-helpers/events')
+const { deployedAddresses } = require('@aragon/templates-shared/lib/ArappFile')(web3)
+const assertRole = require('@aragon/templates-shared/helpers/assertRole')(web3)
+const decodeEvents = require('@aragon/templates-shared/helpers/decodeEvents')
+
+const ACL = artifacts.require('ACL')
+const Kernel = artifacts.require('Kernel')
+const Voting = artifacts.require('Voting')
+const BareTemplate = artifacts.require('BareTemplate')
+
+contract('Bare', ([_, deployer, tokenAddress, authorized]) => {
+  let bareTemplate, dao, acl, voting, CREATE_VOTES_ROLE
+
+  const SUPPORT = 50e16
+  const ACCEPTANCE = 20e16
```

Comment:
Perhaps we could use `pct16()` (although perhaps we could also think of a better name for this utility, like `votePercentage()`)